### PR TITLE
Add GoDoc badge

### PR DESCRIPTION
**Pull request by justincampbell**
To make it easier for users to get to it from this page

### CODE REVIEW

**Comment by bwoskow-ld** [comment]
Hi Justin,

Thanks for the suggestion! This is a good idea.

However, a few months ago we froze `go-client` from future changes when we renamed the project to `go-server-sdk`. See [this note](https://github.com/launchdarkly/go-client/blob/v4/CHANGELOG.md#note-on-future-releases) for information or the [blog post announcement](https://blog.launchdarkly.com/launchdarkly-sdk-update-hard-things-in-computer-science/) for a more complete explanation.

In short, I'm going to decline this pull request, but if you resubmit the pull request against [launchdarkly/go-server-sdk](https://github.com/launchdarkly/go-server-sdk) we will accept it and give you credit for the change.

Cheers,
Ben
